src/cameras/CubeCamera.js

```diff
@@ -87,6 +87,23 @@ function CubeCamera( near, far, cubeResolution ) {
 
 	};
 
+	this.clearCubeMap = function ( color, depth, stencil ) {
```

Comment:
I think I would call it `this.clear = ` instead. Probably should rename `this.updateCubeMap = ` to `this.update = ` too.